src/remarx/app/utils.py

```diff
@@ -7,19 +7,55 @@
 import tempfile
 from collections.abc import Generator
 
-from marimo._cli import cli
+import marimo as mo
+import uvicorn
+from fastapi import FastAPI
 
 # Does this class have a public facing type definition?
 from marimo._plugins.ui._impl.input import FileUploadResults
 
-from remarx.app import ui
+import remarx
 
 
 def launch_app() -> None:
-    """Launch the remarx app into web browser."""
-    with contextlib.suppress(SystemExit):
-        # Prevent program from closing when marimo closes
-        cli.main(["run", ui.__file__])
+    """Launch the remarx app in the default web browser"""
+    # Create marimo asgi app
+    server = mo.create_asgi_app()
+
+    # Add notebooks
+    ## For now set corpus builder as landing page
+    server = server.with_app(path="", root=remarx.app.corpus_builder.__file__)
+    server = server.with_app(
+        path="/corpus-builder", root=remarx.app.corpus_builder.__file__
+    )
```

Comment:
Rather than bind the notebook twice, let's implement the root url as a redirect to the corpus-builder url. We can always add a top level landing page later if we decide to add one.